data/_ui/tooltips.txt

```diff
@@ -448,10 +460,10 @@ tip "reverse thrusting heat:"
 	`Heat produced per second when firing reverse thrusters.`
 
 tip "reverse thrusting shields:"
-	`Shields displaced, consumed, or destroyed, per second when firing reverse thrusters.`
+	`Shields displaced, consumed, or destroyed per second when firing reverse thrusters.`
 
 tip "reverse thrusting hull:"
-	`Hull that flakes off, breaks apart, or is otherwise compromised, per second when firing reverse thrusters.`
+	`Hull that flakes off, breaks apart, or is otherwise compromised per second when firing reverse thrusters.`
```

Comment:
I feel like these should just be simplified to "(Shield || Hull) damage taken per second..." instead of providing explanations for why the shields or hull might be getting used up. Such explanations are what the outfit's description should provide.

Reply:
`Shield damage dealt to the ship per second when firing reverse thrusters.`
Does this look good?

Also, is the accumulative damage for thrust worded properly?

Reply:
That looks good. And I believe "accumulated" is used for various thrust costs because those are the DoT damage types. So when you fire a thruster that has 100 ion/sec, you're not losing 100 energy, you're accumulating 100 units of ionization that will decay over the next few seconds. I suppose that brings into question whether "accumulated" should be used for other firing DoT costs, or if the wording of other firing DoT costs should be used for the engine DoT costs.